src/schemas/movies.py

```
@@ -13,91 +13,101 @@
+    id: int
+    name: str
+    model_config = {"from_attributes": True}
+
+
+class DirectorSchema(BaseModel):
+    id: int
+    name: str
+    model_config = {"from_attributes": True}
 
 
 class MovieBaseSchema(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)